packages/react-components/react-accordion/library/src/components/AccordionHeader/useAccordionHeader.tsx

```diff
@@ -83,7 +84,14 @@ export const useAccordionHeader_unstable = (
     expandIcon: slot.optional(expandIcon, {
       renderByDefault: true,
       defaultProps: {
-        children: <ChevronRightRegular style={{ transform: `rotate(${expandIconRotation}deg)` }} />,
+        children: (
+          <ChevronRightRegular
+            style={{
```

Comment:
I know this is building off the existing `style` implementation but I wonder if this could be implemented as a `className` instead? Should have slightly better perf.

Not blocking though.